Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealityRegisteredServiceProvidersProfile.asset

```diff
@@ -56,7 +56,7 @@ MonoBehaviour:
     runtimePlatform: -1
     configurationProfile: {fileID: 0}
   - componentType:
-      reference: Microsoft.MixedReality.Toolkit.Core.Devices.SpatialAwareness.WindowsMixedRealitySpatialObserver,
+      reference: Microsoft.MixedReality.Toolkit.Core.Services.SpatialAwarenessSystem.BaseSpatialObserver,
```

Comment:
Ah it's only the base spatial observer, need to update this with the actual windows spatial observer.